MauldinEconomics/mautic
# Added Add to DoNotContact Form Action

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? | N
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR adds the Add to DNC as a form action (the Remove Action is already in the system)
![Form Action](https://user-images.githubusercontent.com/9114429/59051752-eda09080-888d-11e9-85bb-ff8aaf67aea3.png)

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create a new form and add the new action along with an email input field
3. Create a new LP that includes this form
4. Go on the LP fill in the email field and submit form
5. Email should be added to the DNC list
